pom.xml

```diff
@@ -1541,9 +1542,7 @@
                     </execution>
                 </executions>
             </plugin>
-        </plugins>
-        <pluginManagement>
-            <plugins>
+
```

Comment:
This is the most tricky bit to me. I'd be wary about modifying maven configuration just for eclipse.